js/character-counter.js

```diff
@@ -6,6 +6,14 @@
 		this.characterCounter.call( el );
 	}
 
+	var length = function( e ){
+		if( /Safari|Chrome/.test(window.navigator.userAgent) ){
```

Comment:
May need to add more browsers here based on where this alternate behavior is needed.